crates/driver/openapi.yml

```diff
@@ -28,7 +28,7 @@ paths:
             - `sell`: amount is in sell_token, out_amount is in buy_token
           schema:
             type: string
-            enum: ["buy", "sell"]
+            enum: [ "buy", "sell" ]
```

Comment:
Out of curiosity how did this get formatted? A bit unfortunate to have these changes in unrelated PRs. Maybe we can have a CI step that verifies that this is formatted correctly. 🤔

Reply:
@MartinquaXD sorry, it is my IDE which does this without my permission :sweat_smile: reverting it.